src/AppInstallerCLICore/Workflows/UninstallFlow.cpp

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "UninstallFlow.h"
+#include "WorkflowBase.h"
+#include "MSStoreInstallerHandler.h"
+#include "ShellExecuteInstallerHandler.h"
+#include "AppInstallerMsixInfo.h"
+
+using namespace AppInstaller::Manifest;
+using namespace AppInstaller::Msix;
+using namespace AppInstaller::Repository;
+
+namespace AppInstaller::CLI::Workflow
+{
+    namespace
+    {
+        std::optional<DWORD> InvokeCreateProcess(const std::string& command, IProgressCallback& progress)
+        {
+            std::wstring commandUtf16 = Utility::ConvertToUTF16(command);
+
+            // Parse the command string as application and command line for CreateProcess
+            wil::unique_cotaskmem_string app = nullptr;
+            wil::unique_cotaskmem_string commandLine = nullptr;
+            THROW_IF_FAILED(SHEvaluateSystemCommandTemplate(commandUtf16.c_str(), &app, &commandLine, NULL));
+
+            STARTUPINFOW startupInfo = { sizeof(startupInfo) };
+            wil::unique_process_information processInfo = {};
+            BOOL success = CreateProcessW(app.get(), commandLine.get(), NULL, NULL, FALSE, 0, NULL, NULL, &startupInfo, &processInfo);
+
+            if (!success)
+            {
+                return GetLastError();
+            }
```

Comment:
THROW_LAST_ERROR_IF() since it's not the actual process exit code?

Reply:
I changed that, but in InvokeShellExecute since this part wasn't needed anymore.